stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<p>I like the use of a hyperlinked outline; very effective.</p><p> </p><p>The formatting's gone a bit askew here though. This is probably just ReviewNB, but worth double checking in the final version.</p><p> </p><p>For the titles of each part, I'd suggest sticking to either Sentence case or Title Case, but not mixing the two.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Fixed (both)</p>